### PR TITLE
Enable Ubuntu's 1710 BuilsTests

### PR DESCRIPTION
**Pull request by jcagme**
After adding Helix Queue for Ubuntu1710 we are enabling BuildTest for this SKU.

This completes: https://github.com/dotnet/core-eng/issues/1307

### CODE REVIEW

File: buildpipeline/pipelines.json [hunk 1]
```diff
@@ -327,7 +327,7 @@
             "HelixJobType": "test/functional/cli/",
             "TargetsWindows": "true",
             "Rid": "win-x64",
-            "TargetQueues": "windows.10.amd64,Windows.10.Amd64.Core",
```
Comment (wtgodbe) [inline buildpipeline/pipelines.json]: Why remove Amd64.Core?

File: buildpipeline/pipelines.json [hunk 2]
```diff
@@ -366,7 +366,6 @@
           },
           "ReportingParameters": {
             "OperatingSystem": "Windows",
-            "Architecture": "arm64",
```
Comment (wtgodbe) [inline buildpipeline/pipelines.json]: Please don't remove this line, it's needed for the arm build-test step.

File: buildpipeline/pipelines.json [hunk 3]
```diff
@@ -385,7 +384,6 @@
...
```
Comment (wtgodbe) [inline buildpipeline/pipelines.json]: Same here

File: buildpipeline/pipelines.json (hunk 2 above)
Comment (jcagme) [inline buildpipeline/pipelines.json]: These changes were not meant to be done. Looks like my local repo was in a weird state (and behind 13030). New commit should be correct

**Comment by jcagme** [comment]
@dotnet-bot test Tizen armel Cross Release Build please

**Comment by wtgodbe** [comment]
This change doesn't affect the CI, so you should be good to merge

**Pull request merged**